core/modules/admin_bar/css/admin_bar.css

```diff
@@ -355,7 +355,7 @@ tr.admin-bar-tweak-permissions-processed td.module {
 /* Alert colors. */
 #admin-bar .admin-bar-alert .error-count {
   box-sizing: content-box;
-  padding: 0.4em 0.5em 0.5em 0.4em;
+  padding: 0.5em 0.4em 0.5em 0.4em;
```

Comment:
Looks like just made an accident when I transposed these values from px to em. This is how it should have been written originally.